web/pgadmin/tools/user_management/__init__.py

```diff
@@ -597,6 +597,9 @@ def delete_user(uid):
         SharedServer.query.filter(SharedServer.servergroup_id.in_(sg)).delete(
             synchronize_session=False)
 
+        # Delete MFA setting for current user.
+        UserMFA.query.filter_by(user_id=uid).delete()
```

Comment:
Shouldn't this be a delete cascade? user id must be having a FK on user table. So it should get deleted automatically on delete in user table.